pkg/tun/device_windows.go

```diff
@@ -142,6 +142,9 @@ $job | Receive-Job
 `, t.interfaceIndex, domain)
 	cmd := dexec.CommandContext(ctx, "powershell.exe", "-NoProfile", "-NonInteractive", pshScript)
 	cmd.DisableLogging = true // disable chatty logging
+	cmd.SysProcAttr = &windows.SysProcAttr{
```

Comment:
Does `powershell.exe` start subprocesses? If not, then why is a process group significant?